server/templates/server/volunteer_table.html

```diff
@@ -21,7 +21,7 @@
             <th scope="col">שם פרטי</th>
             <th scope="col">שם משפחה</th>
             <th scope="col">סוג</th>
-            <th scope="col">גיל</th>
+            <th scope="col">תאריך לידה/גיל</th>
```

Comment:
```suggestion
            <th scope="col">תאריך לידה</th>
```

Reply:
It shows both options, for now I say we keep it.